[2022-12-04T16:13:05.269+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: data_generation.generate_data scheduled__2022-12-03T14:00:00+00:00 [queued]>
[2022-12-04T16:13:05.284+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: data_generation.generate_data scheduled__2022-12-03T14:00:00+00:00 [queued]>
[2022-12-04T16:13:05.286+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-04T16:13:05.291+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2022-12-04T16:13:05.292+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-04T16:13:05.306+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): generate_data> on 2022-12-03 14:00:00+00:00
[2022-12-04T16:13:05.314+0000] {standard_task_runner.py:55} INFO - Started process 336 to run task
[2022-12-04T16:13:05.321+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'data_generation', 'generate_data', 'scheduled__2022-12-03T14:00:00+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/generate_data.py', '--cfg-path', '/tmp/tmplqa9a9h7']
[2022-12-04T16:13:05.329+0000] {standard_task_runner.py:83} INFO - Job 31: Subtask generate_data
[2022-12-04T16:13:05.484+0000] {task_command.py:389} INFO - Running <TaskInstance: data_generation.generate_data scheduled__2022-12-03T14:00:00+00:00 [running]> on host c7dedbf79ea9
[2022-12-04T16:13:05.578+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_generation
AIRFLOW_CTX_TASK_ID=generate_data
AIRFLOW_CTX_EXECUTION_DATE=2022-12-03T14:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-03T14:00:00+00:00
[2022-12-04T16:13:05.909+0000] {python.py:177} INFO - Done. Returned value was: None
[2022-12-04T16:13:05.922+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=data_generation, task_id=generate_data, execution_date=20221203T140000, start_date=20221204T161305, end_date=20221204T161305
[2022-12-04T16:13:05.978+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-12-04T16:13:06.007+0000] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
